azure-pipelines.yml

```diff
@@ -1,314 +1,315 @@
 # Commit triggers
 trigger:
-- master
+  - master
 
 # PR triggers
 pr:
   branches:
     include:
-    - master
+      - master
   paths:
     include:
-    - azure-pipelines.yml
-    - src/*
+      - azure-pipelines.yml
+      - src/*
 
 pool:
-  vmImage: 'windows-latest'
+  vmImage: "windows-latest"

```

Comment:
Is there any difference in this file beyond whitespace and quotes?

Reply:
Oops, not anymore (I'd tried de-duping the "restore" steps, but to no avail) I'll revert this file